Sources_v3/Database/DatabaseContainer.swift

```diff
@@ -189,6 +181,29 @@ class DatabaseContainer: NSPersistentContainer {
                 queue: nil
             ) { log.debug("Data saved to DB: \(String(describing: $0.userInfo))") }
     }
+    
+    /// Load persistent store and recreate it if failed
+    private func setupPersistentStore(shouldFlushOnStart: Bool) throws {
+        var storeLoadingError: Error?
+        
+        loadPersistentStores { _, error in
+            storeLoadingError = error
+        }
+        
+        if (storeLoadingError != nil || shouldFlushOnStart),
```

Comment:
If `shouldFlushOnStart == true` you load a persistent store just to delete it on the next line :)